modules/oe_theme_helper/oe_theme_helper.module

```diff
@@ -76,3 +81,67 @@ function oe_theme_helper_field_widget_oe_paragraphs_variants_form_alter(&$elemen
     $element['subform']['field_oe_icon']['#access'] = FALSE;
   }
 }
+
+/**
+ * Prepares variables for in-page navigation.
+ *
+ * Default template: oe-theme-helper-in-page-navigation.html.twig.
+ *
+ * @param array $variables
+ *   An array of variables.
+ */
+function template_preprocess_oe_theme_helper_in_page_navigation(array &$variables) {
+  $variables['title'] = $variables['element']['#title'];
+  $variables['entity'] = $variables['element']['#entity'];
+  $variables['items'] = $variables['element']['#items'];
+
+  // Process in-page navigation items, assigning them a unique ID.
+  foreach ($variables['items'] as $key => $item) {
+    $variables['items'][$key]['id'] = strtolower(Html::cleanCssIdentifier($variables['items'][$key]['label']));
+  }
+}
+
+/**
+ * Default theme suggestions for custom field groups.
+ *
+ * @param array $variables
+ *   An array of variables passed to the theme hook.
+ * @param string $hook
+ *   Hook name.
+ *
+ * @return array
+ *   Theme suggestions.
+ */
+function _oe_theme_helper_field_group_theme_suggestions(array $variables, string $hook): array {
```

Comment:
We don't need separate function anymore - functionality can be moved to oe_theme_helper_theme_suggestions_oe_theme_helper_in_page_navigation.